### PR TITLE
Rainfall QC

### PR DESCRIPTION
**Pull request by bethanclarke**
This fixes issues outlines by Roger in issue #4649

@shadrackkibet could you please test this but be aware of the data frame merge problem that Danny has pointed out in #4649

The idea is that you will take over this dialog, David will have a chat with you on what needs to be done or I will chat with David and I will outline what needs to be done

### CODE REVIEW

**Comment by shadrackkibet** [comment]
Thanks, i will test this.

**Comment by shadrackkibet** [comment]
I see you have added a button (**omit months**). What should this do?

**Comment by bethanclarke** [comment]
Ah yes, that is for Dry Month so disable for now

**Comment by shadrackkibet** [comment]
The month receiver is autofilled with  Rain column, is this what you want?

**Comment by shadrackkibet** [comment]
@bethanclarke i would like to know what is remaining in this dialog so that i can take over. Could you please highlight them here?

**Comment by bethanclarke** [comment]
Hi @shadrackkibet please could you implement the same changes you’ve made to logical and calculated columns in temp QC for Rain QC

**Comment by bethanclarke** [comment]
I think these changes should be merged for the new release. @shadrackkibet part on logical and calculated columns can wait as these options do work currently

**Comment by dannyparsons** [comment]
@shadrackkibet can you test this?

**Comment by shadrackkibet** [comment]
Testing this.

**Comment by shadrackkibet** [comment]
@dannyparsons this is working favorably well. Please merge it so that i can work on other remaining issues.

File: instat/dlgClimaticCheckDataRain.vb [hunk 1]
```diff
@@ -92,6 +95,8 @@ Public Class dlgClimaticCheckDataRain
         ucrReceiverElement.Selector = ucrSelectorRain
         ucrReceiverElement.SetParameterIsString()
         ucrReceiverElement.bWithQuotes = False
+        ucrReceiverMonth.SetClimaticType("rain")
+        ucrReceiverMonth.bAutoFill = True
```
Comment (dannyparsons) [inline instat/dlgClimaticCheckDataRain.vb]: Should these be element?

Comment (shadrackkibet) [reply instat/dlgClimaticCheckDataRain.vb]: yes, i am going to correct this.

**Pull request merged**